tx_list_test.go

```diff
@@ -701,6 +702,63 @@ func TestTx_LRemByIndex(t *testing.T) {
 	assertions.Equal(0, removedNum, "TestTx_LRemByIndex")
 }
 
+func TestTx_ExpireList(t *testing.T) {
+	InitForList()
+	assertions := assert.New(t)
+	db, err = Open(opt)
+	assertions.NoError(err, "TestTx_ExpireList")
+	tx, err = db.Begin(true)
+
+	bucket := "myBucket"
+	key := []byte("myList")
+	if err := tx.LPush(bucket, key, []byte("d"), []byte("c"), []byte("b"), []byte("a")); err != nil {
+		t.Error("TestTx_ExpireList err")
+	}
+	tx.Commit()
+
+	tx, _ = db.Begin(true)
```

Comment:
db.Begin(true) =>  db.Begin(false) ,coz it's read tx(LRange(bucket, key, 0, -1))